src/components/Text/Text.module.css

```diff
@@ -3,7 +3,7 @@
   font-weight: normal;
   hyphens: auto;
   line-height: var(--leading);
-  color: var(--color);
+  color: var(--text-color);
```

Comment:
In the CSS file, I was using variables to make the specification. However, this time I made the colour specification directly from JavaScript.